hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -182,6 +182,64 @@ public function upgrade_1431() {
 
     return TRUE;
   }
+  
+  /**
+   * Sets case type category to Vacancy for application type or
+   * to Workflow if otherwise
+   *
+   * @return bool
+   */
+  public function upgrade_1432() {
+    $customFieldId = CRM_Core_BAO_CustomField::getCustomFieldID('Category');
+    if ($customFieldId == null) {
+      return FALSE;
+    }
+    
+    $category = "custom_" . $customFieldId;
+    // set category to Vacancy if name is application
+    $categoryValue = $this->getOptionValue('Vacancy');
+    if ($categoryValue['count'] != 0) {
+      civicrm_api3('CaseType', 'get', [
+        'name' => 'application',
+        'api.CaseType.create' => [
+          'id' => '$value.id',
+          $category => $categoryValue['values'][0]['value']
+        ],
+      ]);
+    }
+    
+    // set category to Workflow if name is not application
+    $categoryValue = $this->getOptionValue('Workflow');
+    if ($categoryValue['count'] != 0) {
+      civicrm_api3('CaseType', 'get', [
+        'sequential' => 1,
+        'name' => ['!=' => 'application'],
+        'api.CaseType.create' => [
+          'id' => '$value.id',
+          $category => $categoryValue['values'][0]['value']
+        ],
+      ]);
+    }
+    
+    return TRUE;
+  }
+  
+  /**
+   * Retrieves option group
+   *
+   * @param $option
+   *
+   * @return array
+   */
+  private function getOptionValue($option) {
```

Comment:
Since this method specifically returns option values from the Case Type Category Option Group, I think it is better to rename it to something like `getCaseTypeCategoryOptionValue()`